src/ft2font_wrapper.cpp

```diff
@@ -635,11 +635,10 @@ static PyObject *PyFT2Font_set_text(PyFT2Font *self, PyObject *args, PyObject *k
     size_t size;
 
     if (PyUnicode_Check(textobj)) {
-        size = PyUnicode_GET_SIZE(textobj);
+        size = PyUnicode_GET_LENGTH(textobj);
         codepoints.resize(size);
```

Comment:
maybe simpler and more efficient to just use https://docs.python.org/3/c-api/unicode.html#c.PyUnicode_AsUCS4Copy instead of managing and additional temporay buffer ourselves?

Reply:
My goal here was the minimal fix to get us off to non-deprecated API.  There is some discussion of removing the deprecated macros in 3.11 so I would rather get _something_ in ASAP.